packages/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -509,6 +509,47 @@ describe('CallingClient Tests', () => {
       expect(registerSpy).not.toBeCalled();
     });
 
+    it('Simulate a network flap before initial registration is done should not trigger re-verify registration: Restore Failure', async () => {
+      expect(line.getRegistrationStatus()).toEqual(MobiusStatus.ACTIVE);
+
+      reg.deregister();
+      line.mobiusDeviceId = undefined;
```

Comment:
We are specifically setting mobiusDeviceId as undefined and deregistering the line. This scenario should be where there were no registrations at all